### DISCUSSION TITLE
Client version value is not visible on DevTools console in KoreanBots V2 version.

### DISCUSSION THREAD

**Discussion by kms0219kms** [koreanbots/koreanbots#436] [제안] [answered]
## Reproduction method (재현방법)
> How can I make it happen? (어떻게 하면 발생시킬 수 있나요?)

If you leave the DevTools console on and refresh, the client version value is not visible.

## Expected normal behavior (예상되는 정상적인 동작)
> What if it's normal? (정상이라면 어떻게 되야하나요?)

Would normally be devtools client version, must be visible to the console, and the relevant Guide link shall be activated.

## Problems Occurred (발생한 문제)
> What kind of problems do you have? (어떤 문제가 발생하나요?)

Client version value is not visible on DevTools console in KoreanBots V2 version.

## Client Version (클라이언트 버전)
> Tell me the client version! (클라이언트 버전을 알려주세요!)
 
This is Koreanbots API v2. I can't check the detailed client version :(

Koreanbots API v2입니다. 상세 클라이언트 버전을 확인할 수 없습니다. :(

## Specification (사양)
> Please let me know OS information and browser version! (OS 정보와 브라우저의 버전을 알려주세요!)

**&lt;Computer Information&gt;**
CPU: 1.1 GHz Quad-Core Intel Core i5
RAM: 8GB 3733MHz LPDDR4X
GPU: Intel Iris Plus Graphics 1536 MB

**&lt;Software Information&gt;**
Edition: macOS Big Sur
Version: 11.5
Installation Date: 2021-06-20
OS Build: 20G5042c

**&lt;Browser Information&gt;**
Chrome, Safari, Whale, Firefox for macOS
Chrome, Safari, Whale, Firefox, Edge for Windows

## Check (확인)
* [x]  Are there any overlapping issues? (중복되는 이슈는 없나요?)
* [x]  Is it possible to cause the bug again? (해당 버그를 다시 발생시킬 수 있나요?)

**Comment by kms0219kms** [comment]
왜 클라이언트 버전이 보이지 않는 것이고, 관련 가이드가 삭제 된 상태인가요? 정상이라면 클라이언트 버전을 이슈에 입력하라고 하는 것이 잘못 된 것 아닌가요?

**Reply by kms0219kms** [reply]
재연한 영상: https://github.com/user-attachments/assets/897e09e6-3058-4cd5-9c85-5e0eb8169b4d

**Reply by kms0219kms** [reply]
재생되지 않으면 reply 해주세요.

**Comment by wonderlandpark** [comment]
해당 사항음 다음 릴리즈에서 수정될 예정입니다.

**Comment by wonderlandpark** [comment, marked as answer]
`v2.1.1` 버전으로 릴리즈되었습니다 🎉
